### ISSUE TITLE
Add Grade Block to New Report Page

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#242]
## What problem is the feature trying to solve?
With issue #204, we've identified more useful reporting for accessibility pros. One of the key missing features is a grade that shows a letter grade, developed from the weight of tags of all the alerts in the report. 

## Desired Solution
On a single report page (see #232), the grade would display a single letter grade, added using the weight of each tag associated to the alerts. 100-90% is an A, 89-80% is a B, 79-70% is a C, 69-60% is a D, 59-50% is a F.

Users would be able to toggle this block on and off and set the weights of tags from the report editor page (see #230).

## Technical Details 
- We may need to consider how this feature queries content, as mentioned in #232. 
- We'll use basic styles from [Bootstrap](http://getbootstrap.com)

## Steps to Completion

- [ ] Develop a "dummy" Twig template, using sample data in the controller.
- [ ] Confirm features via User Testing.
- [ ] Make updates from User Testing lessons.
- [ ] Replace sample data with actual features.
- [ ] Integrate page and related features into the core codebase.

**Comment by bbertucc** [comment]
We decided to do away with grading mostly because it adds a bag of worms that we're unprepared to build now.
